tests/forms/Droid/MainActivity.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+
+using Android.App;
+using Android.Content;
+using Android.Content.PM;
+using Android.Runtime;
+using Android.Views;
+using Android.Widget;
+using Android.OS;
+
+namespace forms.Droid
+{
+    [Activity(Label = "forms.Droid", Icon = "@drawable/icon", Theme = "@style/MyTheme", MainLauncher = true, ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation)]
+    public class MainActivity : global::Xamarin.Forms.Platform.Android.FormsAppCompatActivity
+    {
+        bool firstOnCreate = true;
+		bool firstOnStart = true;
```

Comment:
The `bool` on this and the following line should be indented the same amount as the previous line.